modules/pel/peltool/peltool.py

```diff
@@ -201,6 +201,29 @@ def prettyPrint(Mdata: str, desiredSpace: int = 34) -> str:
     return '\n'.join(lines)
 
 
+def excludePEL(uh: UserHeader, config: Config) -> bool:
+    """
+    Evaluates whether a PEL meets criteria based on specified conditions.
+    Returns True if the entry should be Excluded, False otherwise.
+    """
+    if config.critSysTerm and uh.eventSeverity != SeverityValues.critSysTermSeverity.value:
+        return True
+    
+    if config.non_serviceable_only and uh.isServiceable():
+        return True
+    
+    if config.serviceable_only and not uh.isServiceable():
+        return True
+    
+    ## TODO: if severity option present, return False
+    
+    if not config.hidden:
+        if uh.actionFlags & ActionFlagsValues.hiddenActionFlag.value or not uh.isServiceable():
```

Comment:
'hidden' has a specific meaning, that the hidden action flag is set, so let's only check that.